### ISSUE TITLE
Unable to download more posts after updating limit

### ISSUE THREAD

**Issue by jakekingrules** [MalloyDelacroix/DownloaderForReddit#154]
Hey, 

New to github, coding, all of this really. I just started using this and made the mistake of not updating my "post limit" settings in all the necessary places. In other words, I was attempting to download a whole subreddit and got blocked by the default 25 post limit. Well it seems that no matter how many times I update those "post limit" settings, I am unable to get the downloader to recognize that and download more than the original 25 posts. Let me know what you think and thanks for all that you've already done.

**Comment by MalloyDelacroix** [comment]
With the previous download, the date limit filter has been tripped.  To fix this: right click the user/subreddit you are trying to download, then go to settings.  Check the box next to "Date Limit", then set the date to sometime in the past (if you're trying to download as much as possible, its easiest to just change the year to 2006).
